src/main/scala/vaquita/VaquitaTop.scala

```diff
@@ -0,0 +1,146 @@
+package vaquita
+import chisel3._
+import chisel3.util._
+import chisel3.stage.ChiselStage
+
+class VaquitaTop extends Module {
+    val io = IO(new Bundle{
+        val instr = Input(UInt(32.W))
+        val rs1_data = Input(SInt(32.W))
+        val dmemReq = Decoupled(new MemRequestIO)
+        val dmemRsp = Flipped(Decoupled(new MemResponseIO))
+        val hazard_rs1_data_in = Input(UInt(32.W))
+        val vl_rs1_out = Output(UInt(32.W))
+        
+    })
+    implicit val config = new VaquitaConfig {}
+    val de_module = Module(new DecodeStage)
```

Comment:
what's a de_module. correct the variable namings. instead of module, stage would be more correct